src/sqsd.groovy

```diff
@@ -68,8 +68,8 @@ try {
         def messages = sqs.receiveMessage(receiveMessageRequest).getMessages()
         println "Received Messages : " + messages.size()
 
-        // Break when empty
-        if(messages.size() <= 0) break
+        // Break when empty if not running daemonized
+        if(messages.size() <= 0 && config.sqsd.run_daemonized < 1) break
```

Comment:
It stays daemonized by not exiting the `while(true)` loop. An error case would cause it to exit and come down though. Dealing with conditional catches in the loop does seem messy.

I don't follow what you mean by "configure the cron expression in a system property". I've seen [a StackOverflow suggestion](http://stackoverflow.com/a/26958348/400760) to have a Docker CMD script that re-writes a template cron file in `/etc/cron.d`. Is that what you mean?